ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java

```diff
@@ -107,9 +107,10 @@ public IPVCallbackHandler(
 
     public IPVCallbackHandler(ConfigurationService configurationService) {
         var kmsConnectionService = new KmsConnectionService(configurationService);
+        this.cloudwatchMetricsService = new CloudwatchMetricsService(configurationService);
         this.accountInterventionService =
-                new AccountInterventionService(configurationService, newHttpClient());
-        this.cloudwatchMetricsService = new CloudwatchMetricsService();
+                new AccountInterventionService(
```

Comment:
Looks like the changes to IPVCallbackHandler in commit 8e2077f could be squashed into the previous commit